docs/usage/shoot_access.md

```diff
@@ -79,3 +54,30 @@ If you want to use the same OIDC configuration for all your shoots by default, t
 Shoots have to "opt-in" for such defaulting by using the `oidc=enable` label.
 
 For further information on `(Cluster)OpenIDConnectPreset`, refer to [ClusterOpenIDConnectPreset and OpenIDConnectPreset](openidconnect-presets.md).
+
+## Static Token kubeconfig
+
+> **Note:** Static token kubeconfig is deprekated for Shoot clusters using Kubernetes version >= 1.27
```

Comment:
I don't think it is correct to say that this option is deprecated only for Shoots using K8s >= 1.27.
According to my understanding we consider the option as deprecated today no matter which K8s version is used. Simply, starting K8s >= 1.27 we lock the field to false - validations should ensure that the static token is disabled for K8s >= 1.27 Shoots. So both things are different - deprecation != enforced validation.

And when we mark something as deprecated, we have to say what is the alternative.

```suggestion
> **Note:** Static token kubeconfig is deprecated. The [`shoots/adminkubeconfig` subresource](#shootsadminkubeconfig-subresource) should be used instead.
```

I don't know if it makes sense to repeat the K8s 1.27 part as it is already written below.

Reply:
For shoots with Kubernetes < 1.27 the config in the shoot spec for the enableStaticTokenKubeconfig in still considered.

Reply:
IMO, deprecation != default values && enforced validation. Default values && enforced validation are a way we phase out the static token kubeconfig option in a gradual manner.
It is wrong to say "static token kubeconfig is deprecated for Shoots clusters using K8s > 1.27" - it is simply forbidden. Deprecated means "it is not recommended to use it, there is a new/better alternative, but you should be able to still use it if you want but have in mind that it will be removed in the future".